assets/translations/en-us.yaml

```diff
@@ -718,6 +718,14 @@ catalog:
             other { seconds }
           }
       wait: Wait
+    multiCluster:
+      label: This is a Multi-cluster App and it cannot be modified here.
```

Comment:
We shouldn't split localisations like this - not all languages necessarily split in the same was as English. I think you should be able to use the `<nuxt-link>` element in the string and pass the target route as an arg - much line we do with `<a href ` - e.g. see the string `v1ClusterTools.monitoring.description`